src/funk/fd_funk_txn.c

```diff
@@ -612,28 +740,31 @@ fd_funk_txn_merge( fd_funk_t *     funk,
   parent->child_head_cidx   = fd_funk_txn_cidx( FD_FUNK_TXN_IDX_NULL );
```

Comment:
Need to move records from child to parent here

Reply:
That's the value layer to come.  (And actually, they don't need to be moved.  Just the gaddrs to them.)

Reply:
(It's all zero copy.  There should be annotations where the value layer operations will be inserted and what those operations need to be.)

Reply:
See line 579 for the relevant annotation to this.